metadata-ingestion/src/datahub/ingestion/source/looker/lookml_concept_context.py

```diff
@@ -275,8 +275,28 @@ def get_including_extends(
                     f"failed to resolve extends view {extend} in view {self.raw_view[NAME]} of"
                     f" file {self.view_file.absolute_file_path}"
                 )
-            if field in extend_view:
-                return extend_view[field]
+            if attribute_name in extend_view:
+                return extend_view[attribute_name]
+
+        return None
+
+    def get_including_extends(
+        self,
+        field: str,
+    ) -> Optional[Any]:
+
+        if field in [DIMENSIONS, DIMENSION_GROUPS, MEASURES]:
```

Comment:
add a comment about why this is the case - certain fields are merged by extends, whereas others are overridden

Reply:
done